app/src/main/java/org/cru/godtools/ui/tooldetails/ToolDetailsFragment.kt

```diff
@@ -121,6 +123,16 @@ class ToolDetailsFragment() : BasePlatformFragment<ToolDetailsFragmentBinding>(R
             }
         }
     }
+
+    fun openTraining(tool: Tool?, primaryTranslation: Translation?) {
+        // TODO: Open training activity
+        tool?.code?.let { code ->
+            val primaryLanguage = primaryTranslation?.languageCode ?: Locale.ENGLISH
+
+            manifestManager.get().getLatestPublishedManifestLiveData(code, primaryLanguage)
```

Comment:
this won't be necessary for training, the manifest will have already been loaded

Reply:
we actually can probably remove this preloading functionality for all of the tool details activity because we are always loading the manifest to determine if there are tips available.